modules/admin_manual/pages/configuration/user/login_policies.adoc

```diff
@@ -0,0 +1,64 @@
+= Login Policies
+:toc: right
+
+:description: ownCloud provides login policies so admins can allow or reject groups of users to access ownCloud via specific login mechanisms.
+
+== Introduction
+
+{description}
+
+Login policies will emit a `failed login` event if the user isn't allowed to login due to a reject rule.
+
+== Usecases
+
+Login policies can be used to restrict members of particular groups to use only particular login types. This is especially true for guest users as they do not have an ownCloud account and cannot be validated via openidconnect but can login using their email and password.
+
+For example, a desired rule could be that only the groups `guests` and `admin` can login to the instance via username + password, while the rest of the users must login through other mechanisms such as openidconnect.
+
+== Configuration
+
+Login policies are handled via groups that users are members of and maintained via the config.php file. See the xref:configuration/server/config_sample_php_parameters.adoc#order-of-login-policies[Order of login policies] for details.
+
+If no login policy is activated in the `loginPolicy.order` list, ownCloud will work normally. Only registered policies can be activated.
```

Comment:
I don't see a reference about how the policies are registered. This could be confusing.

The registration of the policy is done by the policy / app itself, so unless documented otherwise, from the admin's perspective is automatic.